backend/test/test_lipas_loader.py

```diff
@@ -82,28 +82,72 @@ def test_get_sport_place_point(loader):
     sport_place = loader.get_sport_place(76249)
     assert sport_place["geom"] == "MULTIPOINT (27.2258867781278 63.545014556221)"
     assert sport_place["season"] == "Talvi"
+    assert sport_place["table"] == "luistelukentta"
 
 
 def test_get_sport_place_line(loader):
     sport_place = loader.get_sport_place(513435)
```

Comment:
Is it so that these lipas types to test are selected randomly?

Reply:
Joona selected this point for test. I just added a check that it is indeed "luistelukentta" without ä, and earlier added a check that it is a Talvi place.

If you have better lipas points to check, feel free to suggest what are the most important ones to check that are imported right?

Reply:
I see :)

The only relevant options at this point might be that we check that
1) objects of the random point / line type class get imported right
2) objects of all type class get imported right (this is probably overkill solution)

Since you now also test that does the import procedure work for veneilyn palvelupaikka objects, we have all seasons also represented in our tests. Let's leave this as it is.